File: week3-app-development/python/Ex6e_Remove_A_Todo.py
#!/usr/bin/env python3
from connection import session

print('========================================')
print('Start exercise')
try:
    session.execute(
        "DELETE FROM todoitems WHERE user_id='john' AND item_id=22222222-5cff-11ec-be16-1fedb0dfd057;")
    session.execute(
        "DELETE FROM todoitems WHERE user_id='mary' AND item_id=33333333-5cff-11ec-be16-1fedb0dfd057;") 
    output = session.execute(
        "SELECT toTimestamp(item_id), completed, title FROM todoitems WHERE user_id = 'john';")
    for row in output:
        print(row)
except Exception as e:
    print(e)
    print('Failure')
else:
    print('Success')
print('========================================')

File: week3-app-development/javascript/Ex6e_Remove_A_Todo.js
const { client } = require("./connection");

console.log("========================================");
console.log("Start exercise");

(async () => {
  try {
    await client.execute(
      "DELETE FROM todoitems WHERE user_id='john' AND item_id=11111111-5cff-11ec-be16-1fedb0dfd057;"
    );
    const result = await client.execute(
      "SELECT toTimestamp(item_id), completed, title FROM todoitems WHERE user_id = 'john';"
    );
    result.rows.forEach((row) => {
      console.log(row);
    });
    console.log("SUCCESS");
  } catch (e) {
    console.log(e);
  }
  console.log("========================================");
  process.exit();
})();
